examples/src/main/proto/echo.proto

```diff
@@ -0,0 +1,44 @@
+// Copyright 2023 The gRPC Authors
```

Comment:
Where did this file come from? This is not from grpc-go, nor grpc main repo.

Note that I'm adding a copy from grpc-go in https://github.com/grpc/grpc-java/pull/9962 , and it is incompatible with what is here.

Reply:
It came from https://github.com/grpc/grpc-go/blob/master/examples/features/proto/echo/echo.proto which I then modified to options and package based on the HelloWorld example.

Reply:
Removed my version of echo.proto